tests/core/tools/test_runner/test_yaml_runner.py

```diff
@@ -14,71 +14,79 @@
 
 class TaxBenefitSystem:
     def __init__(self):
-        self.variables = {'salary': TestVariable()}
+        self.variables = {'salary': MyVariable()}
         self.person_entity = Entity('person', 'persons', None, "")
         self.person_entity.set_tax_benefit_system(self)
 
-    def get_package_metadata(self):
+    @staticmethod
+    def get_package_metadata():
         return {"name": "Test", "version": "Test"}
 
-    def apply_reform(self, path):
+    def apply_reform(self, _path):
         return Reform(self)
 
-    def load_extension(self, extension):
+    @staticmethod
+    def load_extension(_extension):
         pass
 
-    def entities_by_singular(self):
+    @staticmethod
+    def entities_by_singular():
         return {}
 
-    def entities_plural(self):
+    @staticmethod
+    def entities_plural():
         return {}
 
     def instantiate_entities(self):
         return {'person': Population(self.person_entity)}
 
-    def get_variable(self, variable_name, check_existence = True):
-        return self.variables.get(variable_name)
+    def get_variable(self, variable_name, check_existence = False):
+        return self.variables.get(variable_name, check_existence)
 
-    def clone(self):
+    @staticmethod
+    def clone():
         return TaxBenefitSystem()
 
 
 class Reform(TaxBenefitSystem):
     def __init__(self, baseline):
+        super().__init__()
         self.baseline = baseline
 
 
 class Simulation:
     def __init__(self):
         self.populations = {"person": None}
 
-    def get_population(self, plural = None):
+    @staticmethod
+    def get_population(plural = None):
         return None
 
 
-class TestFile(YamlFile):
+class MyFile:
 
     def __init__(self):
         self.config = None
+        self.nodeid = 'testname'
         self.session = None
-        self._nodeid = 'testname'
 
 
-class TestItem(YamlItem):
+class MyItem(YamlItem):
```

Comment:
I understand we have a naming conflict with the `pytest` API. However, these new names are not satisfactory.
What about `Stub` or `Fixture`? 🙂